devbuild.sh

```diff
@@ -112,6 +115,8 @@ while :; do
     --app|--app=) usage_error "$1 requires argument." ;;
     --ccpp=?*) CCPP=${1#*=} ;;
     --ccpp|--ccpp=) usage_error "$1 requires argument." ;;
+    --rrfs) RRFS="on" ;;
```

Comment:
For consistency with the other "boolean" variables (`CONTINUE`, `VERBOSE`, etc), can we make the valid values for this `true|false`?  I guess that will require `-DENABLE_RRFS` in the setting of `CMAKE_SETTINGS` below to `new` instead of `${RRFS}`.

Reply:
I would prefer not to do that because the idiomatic way of using CMake boolean options is by using "on" or "off".  For CMake options which use specific values, those get set in a slightly different way that is more complicated, so changing it would not just deviate from established standards for how CMake boolean options are treated, but would also require more complicated CMake code.  Keeping consistency with common established usage patterns for CMake options is easier to implement, and is more important in my opinion because it gives users the expected experience rather than a custom one-off one that is different than all other CMake projects users may encounter.  The `CONTINUE` and `VERBOSE` are not CMake options, so having those be `true|false` doesn't matter as much since there isn't a particular standard to adhere to for those.  I am not following your comment about `new`.  I don't understand what you're referring to there.

Reply:
That `new` was a typo, I meant `yes` or `no`, depending on whether `RRFS` is `true` or `false` (result of multitasking).  So you'd have below something like `if RRFS = true then -DENABLE_RRFS=yes else -DENABLE_RRFS=${RRFS}=no fi`.  But it's not a big deal.